generated/nidaqmx/constants.py

```diff
@@ -868,23 +774,24 @@ class TimestampEvent(Enum):
 
 class TorqueUnits(Enum):
     NEWTON_METERS = 15881  #: Newton meters.
-    FOOT_POUNDS = 15884  #: Pound-feet.
-    INCH_POUNDS = 15883  #: Pound-inches.
     INCH_OUNCES = 15882  #: Ounce-inches.
+    INCH_POUNDS = 15883  #: Pound-inches.
+    FOOT_POUNDS = 15884  #: Pound-feet.
     FROM_CUSTOM_SCALE = 10065  #: Units a custom scale specifies. If you select this value, you must specify a custom scale name.
 
 
 class TriggerType(Enum):
-    NONE = 10230  #: Disable reference triggering for the task.
```

Comment:
The old docstring was incorrect because it referred to reference triggering. The new docstring is correct.